test/components/reconciliation-reports/reconciliation-report.js

```diff
@@ -11,12 +11,50 @@ configure({ adapter: new Adapter() });
 
 const reconciliationReports = {
   map: {
-    exampleReport: {
+    exampleInventoryReport: {
       data: {
         reportStartTime: '2018-06-11T18:52:37.710Z',
         reportEndTime: '2018-06-11T18:52:39.893Z',
         status: 'SUCCESS',
         error: null,
+        type: 'Inventory',
+        okFileCount: 21,
+        filesInCumulus: {
+          okCount: 129,
+          onlyInS3: [
+            's3://some-bucket/path/to/key-1.hdf',
+            's3://some-bucket/path/to/key-2.hdf'
+          ],
+          onlyInDynamoDb: [
+            {
+              uri: 's3://some-bucket/path/to/key-123.hdf',
+              granuleId: 'g-123'
+            },
+            {
+              uri: 's3://some-bucket/path/to/key-456.hdf',
+              granuleId: 'g-456'
+            }
+          ],
+        },
+        collectionsInCumulusCmr: {
+          okCount: 1
+        },
+        granulesInCumulusCmr: {
+          okCount: 7
+        },
+        filesInCumulusCmr: {
+          okCount: 4
+        }
+      }
+    },
+    // TODO: change this once we have the actual GNF format
```

Comment:
remove the TODO?

Reply:
Nice catch! https://github.com/nasa/cumulus-dashboard/pull/840/commits/63672daedb9427b2e75f04f50608d770fc3aff72